supplementary_style_guide/glossary_terms_conventions/general_conventions/o.adoc

```diff
@@ -164,6 +164,51 @@
 
 *See also*:
 
+[discrete]
+[[operator]]
+==== image:images/yes.png[yes] Operator (noun)
+*Description*: In the context of Kubernetes, an Operator is a method of packaging, deploying, and managing a
+Kubernetes application. A Kubernetes application is an application that is both
+deployed on a Kubernetes cluster (including OpenShift clusters) and managed
+using the Kubernetes APIs and `kubectl` or `oc` tooling.
+
+The term "Operator" in the context of Kubernetes is always capitalized to distinguish it from other types of operators, such as human or mathematical operators. For example:
+
+----
+= Support policy for unmanaged Operators
+
+Individual Operators have a `managementState` parameter in their configuration.
+----
+
+----
```

Comment:
If you want to include an example of a place that uses a lowercase operator, I'd suggest adding a heading to each of these examples. Something like:

```
.Example: Kubernetes Operator
----
= Support policy for unmanaged Operators
Individual Operators have a `managementState` parameter in their configuration.
----

.Example: Mathematical operator
----
The following operators and operands are supported in Drools Rule Language:
* + (addition)
* - (subtraction)
...
----
```

**However** I'm wondering if it makes sense to break out the lowercase "operator" usage into its own term entry. Because it's slightly confusing to mix them here, because the "Incorrect forms" field lists "operator" as an incorrect entry.